aviary/subsystems/propulsion/test/test_turboprops.py

```diff
@@ -193,7 +190,7 @@ def test_case_4(self):
         filename = get_path('models/engines/turboprop_1120hp.deck')
         test_points = [(0, 0, 1)]
         point_names = ['SLS',]
-        truth_vals = [(1120, 136.3, 644),]
+        truth_vals = [(1120, 136.3, -644),]
         # test_points = [(0, 0, 0), (0, 0, 1), (.6, 25000, 1)]
         # point_names = ['idle', 'SLS', 'TOC']
         # truth_vals = [(112, 0, 195.8), (1120, 0, 644), (1742.5, 0, 839.7)]
```

Comment:
the single truth val was only being used because num_nodes > 1 wasn't supported at the time, can you change this and the other test to use the three points?

Reply:
Good call, I flipped cases 3 and 4 to using the vectorized set of data and made some corresponding changes